doc/code_snippets/test/default_values/default_functions_test.lua

```diff
@@ -0,0 +1,65 @@
+local fio = require('fio')
+local server = require('luatest.server')
+local t = require('luatest')
+local g = t.group()
+g.before_each(function(cg)
+    cg.server = server:new {
+        box_cfg = {},
+        workdir = fio.cwd() .. '/tmp'
+    }
+    cg.server:start()
+end)
+
+g.after_each(function(cg)
+    cg.server:drop()
+    fio.rmtree(cg.server.workdir)
+end)
+
+g.test_default_func = function(cg)
+    cg.server:exec(function()
+
+        -- create_no_arg_function_start
+        box.schema.func.create('current_year', {
+            language = 'Lua',
+            body = "function() return require('datetime').now().year end"
+        })
+        -- create_no_arg_function_end
+
+        -- format_space_default_func_start
+        local books = box.schema.space.create('books')
+        books:format({
+            { name = 'id', type = 'unsigned' },
+            { name = 'isbn', type = 'string' },
+            { name = 'title', type = 'string', default = 'No title specified' },
```

Comment:
Just an idea: as we talk about default functions here, we can remove the `default` parameter for `title` to simplify the example a bit.